### PR TITLE
feat(create-collector-step3): add step3 page component & AttachedServiceAccountForm

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- Added new state to store(collector-form-store)
- Added CreateCollectorStep3 markup
- Added AttachedServiceAccountForm components markup & little function

Dynamic field (JSON schema) has not yet been implemented.

image
image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/store/collector-form-store.ts [hunk 1]
```diff
@@ -10,10 +13,15 @@ type CollectorFormAction = any;
...
     state: () => ({
         originCollector: null,
+        attachedServiceAccount: null,
     }),
     actions: {
         setOriginCollector(collector: any) {
             this.originCollector = collector;
         },
+        setAttachedServiceAccount(serviceAccount: AttachedServiceAccount) {
+            if (!serviceAccount || !serviceAccount.length) this.attachedServiceAccount = null;
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/store/collector-form-store.ts:23]: nitpick: I think we can shorten this. :)

```suggestion
            if (!!serviceAccount?.length) this.attachedServiceAccount = null;
```

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep3.vue [hunk 2]
```diff
@@ -1,8 +1,131 @@
...
+    emit('update:currentStep', 4);
+};
+const handleClickPrevButton = () => {
+    emit('update:currentStep', 2);
+};
+
+const handleClose = () => {
+    emit('update:currentStep', 1);
+};
+
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep3.vue:91]: praise: Nice clean codes!

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+        name: 'all',
+    },
+    {
+        label: 'Specific Service Account', // TODO: translation
+        name: 'specific',
+    },
+];
+
+const state = reactive({
+    selectedAttachedServiceAccountType: 'all',
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue:79]: suggestion: Please add types too!

```suggestion
    selectedAttachedServiceAccountType: 'all' as 'all'|'specific',
```

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    {
+        name: '82f0bd12-a630-4307-b9a3-d44f867a72d0',
+        label: 'vortals',
+        type: 'item',
+    },
+    {
+        name: '8845d702-4f89-478e-9227-d90b33c42a60',
+        label: 'Lempira',
+        type: 'item',
+    }],
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue:94]: suggestion: Please add types too!

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+                         @change="handleChangeAttachedServiceAccountType"
+                >
+                    {{ item.label }}
+                </p-radio>
+            </p-radio-group>
+            <p-select-dropdown class="attached-service-account-dropdown"
+                               :selected="state.selectedAttachedServiceAccountType"
+                               :items="attachedServiceAccountList"
+                               @update:selected="handleChangeAttachedServiceAccountType"
+            />
```
Comment (WANZARGEN) [inline apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue:25]: nitpick: Can you add comments that they are shown optionally by device width? :)

**Review by WANZARGEN — APPROVED** [review]
Overally, LGTM!!!
Nice clean codes :)
I added very few minor comments!

File: apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep3.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/services/asset-inventory/collector/create-collector/modules/CreateCollectorStep3.vue:91]: I think so too!!!

**Review by yuda110 — APPROVED** [review]
LGTM. Nice commit splitting 👍 Very easy to read.

**Pull request merged**